webapp/src/components/AssetPage/SaleRentActionBox/SaleRentActionBox.tsx

```diff
@@ -104,7 +104,7 @@ const SaleRentActionBox = ({
   const handleOnRent = useCallback(() => {
     if (!!authorization && hasAuthorization(authorizations, authorization)) {
```

Comment:
```suggestion
    if (!!authorization && hasAuthorization(authorizations, authorization) && selectedRentalPeriodIndex !== undefined) {
```
what if we add this check here so we avoid the ternaries from below? 
like: `onRent(selectedRentalPeriodIndex ? selectedRentalPeriodIndex : 0)`, it would turn back to `onRent(selectedRentalPeriodIndex)`